appengine/standard_python37/pubsub/main.py

```diff
@@ -40,20 +40,21 @@
 TOKENS = []
 HEADERS = []
 CLAIMS = []
+REQUEST_HEADERS = []
 
 # [START index]
 @app.route('/', methods=['GET', 'POST'])
 def index():
     if request.method == 'GET':
-        return render_template('index.html', messages=MESSAGES, tokens=TOKENS, headers=HEADERS, claims=CLAIMS)
+        return render_template('index.html', messages=MESSAGES, tokens=TOKENS, headers=HEADERS, claims=CLAIMS, request_headers=REQUEST_HEADERS)
 
     data = request.form.get('payload', 'Example payload').encode('utf-8')
 
     publisher = pubsub_v1.PublisherClient()
     topic_path = publisher.topic_path(app.config['GCLOUD_PROJECT'],
                                       app.config['PUBSUB_TOPIC'])
-    publisher.publish(topic_path, data)
-
+    future = publisher.publish(topic_path, data)
+    future.result()
```

Comment:
Sanity check - are we fine with letting any exception here to simply bubble up, or do we want to explicitly handle it?

Reply:
I didn't do publish with error handling here. The idea would be to add `future.add_done_callback(callback)`. I think it's okay to let the exception bubble up here.